modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialogList.ts

```diff
@@ -20,7 +20,7 @@ module app.create {
             });
         }
 
-        private notifySelected(listItem: NewContentDialogListItem) {
```

Comment:
It is really needed bacause we really have to override parent's createItemView()